api/tests/connectors/acceslibre/acceslibre_test.py

```diff
@@ -204,3 +204,18 @@ def test_should_raise_error_when_activity_is_not_a_dict(self, requests_mock):
         with pytest.raises(acceslibre.AccesLibreApiException) as exception:
             acceslibre.get_id_at_accessibility_provider(name=name, public_name=public_name, ban_id=ban_id)
         assert str(exception.value) == "Acceslibre activite should be a dict, but received: Chaîne de caracteres"
+
+    def test_if_id_exists_at_acceslibre(self, requests_mock):
+        existing_slug = "bibliotheque-municipale-de-truchan-les-bains"
+        unexisting_slug = "sniab-sel-nahcurt-ed-elapicinum-euqehotlibib"
```

Comment:
J'imagine que tu voulais faire ça :
```suggestion
        unexisting_slug = "sniab-sel-nahcurt-ed-elapicinum-euqehtoilbib"
```
ou juste voir si j'allais réagir :rofl: 

(pas besoin de corriger à moins que l'inversion finisse par te choquer !)

Reply:
faute de frappe en effet. Mais maintenant que je l'ai vu, je ne peux plus ne pas le voir 😅